qemu/tests/cfg/hv_avic.cfg

```diff
@@ -7,5 +7,6 @@
     cpuid_chk_cmd = "rpm -qa | grep cpuid"
     cpuid_pkg = "cpuid"
     RHEL:
-        repo_install_cmd = "dnf install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-%s.noarch.rpm"
+        #set workaround from RHEL10.0 since no cpuid package includes in epel-10.repo
+        repo_install_cmd = "dnf install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-9.noarch.rpm"
```

Comment:
@peixiu QEMU 6.2 is available for RHEL8,  do you plan to remove the rhel8 support ?

Reply:
@yanan-fu yes, our testing focus on the recent rhel9 and rhel10 version.

Reply:
@peixiu  This test case configuration does not have the limitation of exclude RHEL-8.